Sources/web3swift/KeystoreManager/IBAN.swift

```diff
@@ -11,6 +11,12 @@ public struct ICAP {
     public var asset: String
     public var institution: String
     public var client: String
+
+    public init(asset: String, institution: String, client: String) {
```

Comment:
This struct is not used anywhere in the project so I assumed it should be used in other projects that use web3swift as a dependency. Thus, this struct must provide an initializer.